### DISCUSSION TITLE
Remote control stops working after 5-10 seconds from remote networks

### DISCUSSION THREAD

**Discussion by xbyte99** [rustdesk/rustdesk#8598] [General]
I have a very curious case where remote controlling one specific Windows 11 PC stops working after 5 seconds. Interestingly this only happens when connecting from any network outside the LAN where this PC is located. Within the LAN, remote controlling this PC works flawlessly from any other device (other Windows PCs, Android phones). Bandwidth from the networks outside is also not an issue and controlling other Windows PC in the same LAN like the affected PC also works from outside networks.

During the first 5 seconds after the initial image is received, all runs smoothly and then starts to lag extremely until there is barely any reaction anymore on the remote controlling device.
On the affected Windows 11 PC however, the remote controlled mouse cursor continues to move across the screen for some more seconds but this is not reflected on the remote controlling device. Until even that movement stops. On the affected Windows 11 PC everything runs fine, Task manager shows very little workload, controlling the mouse, keyboard, applications works fine.

I've tried a lot of things, been through all combinations of RustDesk versions from 1.2.2 up to 1.2.6 in client and server side. Tried different video codecs. Disabled hardware code, adaptive bitrate, capture screen using DirectX. To no avail. I'm out of ideas

Unfortunately this specific Windows 11 PC is my main machine with which I work most of the time. There was no issue with remote controlling this machine before, all this started maybe a few weeks ago.
RustDesk Server is hosted on my Synology NAS with the latest version as a docker container. Maybe the issue occurred when I updated to the newest RustDesk Server version but I cannot tell for sure.

So if anyone here has an idea what might cause this weird behavior, please enlighten me! Are there any logs/debug options I can enable/look into?
Thanks a lot!

**Comment by istvanmate** [comment]
Just had a similar issue. Tried to connect to an external machine, which has no uPnP or port forwarding, from a location, which is probably behind a double NAT connection. Not sure, which is the issue, but for me the Always connect via relay checkbox fixed it.
